### ISSUE TITLE
RGI database manager fails to find a module

### ISSUE THREAD

**Issue by apetkau** [arpcard/rgi_wrapper#3]
# Issue

I tested out the most recent RGI database manager tool and I found it fails with the message:

```
Fatal error: Exit code 1 (Error)
Traceback (most recent call last):
  File "/home/aaron/software/shed_tools/jupiter.cscscience.ca/repos/admin/rgi/feea0bca16b1/rgi/data_managers/data_manager_rgi_build_db/data_manager/rgi_database_builder.py", line 10, in <module>
    from import_data import run, _main
  File "/home/aaron/software/shed_tools/jupiter.cscscience.ca/repos/admin/rgi/feea0bca16b1/rgi/data_managers/data_manager_rgi_build_db/data_manager/import_data.py", line 10, in <module>
    import rgi.database.write_fasta_from_json
ImportError: No module named 'rgi'
```

It's specifically this line that it fails at:

https://github.com/arpcard/rgi_wrapper/blob/master/data_managers/data_manager_rgi_build_db/data_manager/import_data.py#L10

I took a look at why it cannot find the RGI modules (since we are depending on the rgi conda environment) and I traced it down to how Galaxy loads up environments for database manager tools:

```
export PATH=$PATH:'/home/aaron/software/galaxy/database/dependencies/_conda/envs/__rgi@4.2.2/bin' ; python3 /home/aaron/software/shed_tools/jupiter.cscscience.ca/repos/admin/rgi/feea0bca16b1/rgi/data_managers/data_manager_rgi_build_db/data_manager/rgi_database_builder.py --out "/home/aaron/software/galaxy/database/files/000/dataset_61.dat
" --name "safasdf" --url "https://card.mcmaster.ca/latest/data"
```

Before the tool is run, the line `export PATH=$PATH:'/home/aaron/software/galaxy/database/dependencies/_conda/envs/__rgi@4.2.2/bin'` is used to add the `rgi` conda environment to the end of `PATH`.  For regular tools it will `source` this environment, which will do much more than add to the PATH. In this case, though, since it's just adding to the `PATH` at the very end, the next part:

```
python3 /.../rgi_database_builder.py
```

May (or may not) run the right version of `python3` (if you have a system-wide version it will prefer this instead).  This means that Python may (or may not) load up the correct RGI modules, which is happening in my case.

# Solutions

The easiest solution is to likely just do what was being done before, that is include a copy of `write_fasta_from_json` in the `import_data.py` file (making sure it's using the most recent version of the code).

Another solution is to look a bit further into how to load up Python environments for database managers in Galaxy, but I'm not sure if there will be any easy way to do this (so this first solution is probably better).

What do you think @raphenya @mjram0s?

**Comment by mjram0s** [comment]
@apetkau Thanks for catching this. I was able to recreate the error. It looks like my tool did not update properly during testing, so I was running the version without the import statement. 

I will include a copy of `write_fasta_from_json` in `import.py` for now until I can look into the environment solution.

**Comment by mjram0s** [comment]
See #5 for changes

**Comment by apetkau** [comment]
Fixed in #5
